src/build/output-permalink.js

```diff
@@ -37,23 +37,24 @@ const {defaultLocale} = require('../site/_data/site');
 module.exports = (data) => {
   const defaultLocaleRegExp = new RegExp(`^/${defaultLocale}/`);
   const localization = data.lang !== defaultLocale;
-
+  let permalink;
   // In eleventy you can set permalink: false to tell it to not output a page.
   // If a page has a false permalink we should completely ignore it.
   if ('permalink' in data && data.permalink === false) {
-    return;
+    return permalink;
   }
 
   if (data.permalink) {
-    return data.permalink.replace(
+    permalink = data.permalink.replace(
       /^\/{{lang}}/,
       localization ? '/i18n/{{lang}}' : '',
     );
+  } else {
+    permalink = data.page.inputPath
```

Comment:
What doest this do? The default permalink doesn't work and it needs to be provided?